internal/exec/util/passwd.go

```diff
@@ -131,10 +133,27 @@ func translateV2_1UsercreateGroupSliceToPasswdUserGroupSlice(groups []types.User
 }
 
 func (u Util) CheckIfUserExists(c types.PasswdUser) (bool, error) {
-	code, err := u.LogCmd(exec.Command(distro.ChrootCmd(), u.DestDir, distro.IdCmd(), c.Name),
-		"checking if user %q exists", c.Name)
+	// code, err := u.LogCmd(exec.Command(distro.ChrootCmd(), u.DestDir, distro.IdCmd(), c.Name),
+	// "checking if user %q exists", c.Name)
+
+	// fmt.Println("ERROR START")
+	// fmt.Println(err.Error())
+
+	// fmt.Println("ERROR ENDDD")
+	code := -1
+	cmd := exec.Command(distro.ChrootCmd(), u.DestDir, distro.IdCmd(), c.Name)
+	stdoutStderr, _ := cmd.CombinedOutput()
```

Comment:
[`cmd.CombinedOutput()`](https://golang.org/pkg/os/exec/#Cmd.CombinedOutput) runs the command, you can drop the `.Run()` call and change the `_` on this line to `err`